### ISSUE TITLE
Email Subscriptions: 2 Column layout displays horridly in Email

### ISSUE THREAD

**Issue by tehjaymo** [Automattic/jetpack#20694]
Hi all this is my first issue. Would appreciate some handholding if i'm missing info! 

E-mail subscribers are receiving a wall of unreadable vertical text when using columns block in post.

#### Steps to reproduce the issue

1. Create a post
2. Add columns block with 2 columns (50/50)
3. Add content to left column Image / text and content/text in right hand column.
4. Publish post and check received subscriber email 

#### What I expected

Either 2 column layout will be respected or (preferred) columns should 'stack on mobile' format like 

#### What happened instead

Left hand column will be nearly full width of the email, right hand column width with be squished to 1 or 2 characters resulting in huge wall of vertical text

Example with Image in left column 

<img width="1260" alt="Screenshot 2021-08-15 at 07 51 59" src="https://user-images.githubusercontent.com/2405970/129469841-b769dd25-b6d8-49b1-8737-36cfe3c758d7.png">
<img width="1225" alt="Screenshot 2021-08-15 at 07 51 09" src="https://user-images.githubusercontent.com/2405970/129469848-3d9863ed-1ec6-4d0a-8690-dcb7dbfb6f44.png">

Example H1 in left column 

<img width="762" alt="Screenshot 2021-08-15 at 07 51 33" src="https://user-images.githubusercontent.com/2405970/129469880-81083962-eafd-4655-8d9d-76e33c6ed0ca.png">

Note: I haven't tested with more than two columns.

**Comment by kathrynwp** [comment]
@jeherve This is still a painful issue, even in the newly redesigned emails. Here's an example:

https://user-images.githubusercontent.com/7662449/170039114-448f3ccd-915c-4d1f-a459-c43c23ddc83e.mov

For comparison, here are the 50%/50% columns on the site itself::

![Weekly_Comet_Update_–_Comet](https://user-images.githubusercontent.com/7662449/170039278-607a9921-1f75-47a5-b05e-ab7ba226f524.jpg)

**Comment by ivan-ottinger** [comment]
Another report: 5294023-zd-woothemes

**Comment by ivan-ottinger** [comment]
I have added the issue to our project board so we can keep track of it. At the same time, it is not clear when our team will be able to look into it.

**Comment by TimBroddin** [comment]
I took a look at it, but in the new templates this seems to have been fixed. However, I noted that something's wrong with the vertical alignment. I opened [a new issue](https://github.com/Automattic/wp-calypso/issues/68577) for this.

<img width="746" alt="CleanShot 2022-10-04 at 11 02 58@2x" src="https://user-images.githubusercontent.com/528287/193803622-3b05d90f-a11e-4f66-8385-cc89ca5a1a1c.png">

I'm going to do some more testing on this ticket, to see if I can make the new template break.
